Curve-finance-suite/Unkill/README.md

```diff
@@ -0,0 +1,20 @@
+# UnkillMe Agent
+
+## Description
+
+This agent detects when Curve Pool Owner calls `unkill_me` functions.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
```

Comment:
I am unsure if this line a leftover from the README or something that should stay

Reply:
Should be removed :+1: